### DISCUSSION TITLE
Widgets auf halbe Breiten ziehen

### DISCUSSION THREAD

**Discussion by Hohenbostler** [Zefau/ioBroker.jarvis#1027] [Q&A]
Ich nutze V3 in der Alpha 80

und möchte gerne die Widgets teilweise auf halbe breite ziehen.

![grafik](https://user-images.githubusercontent.com/42488069/134290554-42bae046-9378-457d-bbca-73f823310f64.png)

Also zum verständnis. Die oben 3 wären so richtig. die unteren 2 aber im prinzip immer 1,5 breite der oberen. Ist so etwas möglich ?

**Comment by Zefau** [comment]
Ja das geht. Einfach entsprechend hinziehen / groß ziehen

**Comment by Hohenbostler** [comment]
Mh sorry ich kann bei size nur 1sw oder 2sw sehen beim ziehen. 1.5 zB nicht

![grafik](https://user-images.githubusercontent.com/42488069/134292873-7506b4f3-8611-4f0b-b51c-4729dd5d34fd.png)

**Comment by mcuiobroker** [comment]
![v3-80-1-5-fache breite](https://user-images.githubusercontent.com/84818934/134292963-69edf4bf-4b5d-4b4c-a554-e24fe24bd826.gif)

**Reply by Hohenbostler** [reply]
Richtig aber auch bei dir steht  Size 1w und 2w. Ich müsste bei mir auf 1.5w kommen
und bei dir sieht man auch wie er springt. Und genau den zwischenraum den bräuchte ich

**Reply by Zefau** [reply]
Jetzt versteh ich  Jo geht nur volle Längen.

**Reply by Hohenbostler** [reply]
Ja stimmt. Kann man das als Feature requests aufnehmen ?
